cmd/run.go

```diff
@@ -59,12 +53,18 @@ https://iver-wharf.github.io/#/usage-wharfyml/
 			}
 			return errors.New("failed to parse .wharf-ci.yml")
 		}
-		res, err := b.Build(context.Background(), def, worker.BuildOptions{
+		log.Info().Message("Successfully parsed .wharf-ci.yml")
+		b, err := worker.NewK8s(context.Background(), def, ns, kubeconfig, worker.BuildOptions{
 			StageFilter: runFlags.stage,
 		})
 		if err != nil {
 			return err
 		}
+		log.Debug().Message("Successfully created builder. Starting build.")
```

Comment:
nit: Could be split up into two logs I think, for finer granularity.
```suggestion
		log.Debug().Message("Successfully created builder.")
		log.Info().Message("Starting build.")
```